dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/DefaultDhisConfigurationProvider.java

```diff
@@ -117,7 +117,7 @@ public void init()
 
         try ( InputStream jsonIn = locationManager.getInputStream( GOOGLE_AUTH_FILENAME ) )
         {
-            Map<String, String> json = new ObjectMapper().readValue( jsonIn, new TypeReference<HashMap<String,Object>>() {} );
+            HashMap<String, Object> json = new ObjectMapper().readValue( jsonIn, new TypeReference<HashMap<String,Object>>() {} );
```

Comment:
Is HashMap intentional?

Reply:
Yeah, there was a generic change in the upgrade in Jackson version... we might be able to fix it later, but it might require a change in code